### PR TITLE
Fix Mesh.canvasPadding

### PR DESCRIPTION
**Pull request by dev7355608**
##### Description of change

The math is flawed: https://www.pixiplayground.com/#/edit/i57Nv699UIsCI3y5wczPd. Fixed it.

##### Pre-Merge Checklist

- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: packages/canvas/canvas-mesh/src/CanvasMeshRenderer.ts [hunk 1]
```diff
@@ -148,34 +148,40 @@ export class CanvasMeshRenderer
...
             const centerY = (y0 + y1 + y2) / 3;
 
             let normX = x0 - centerX;
             let normY = y0 - centerY;
 
-            let dist = Math.sqrt((normX * normX) + (normY * normY));
-
-            x0 = centerX + ((normX / dist) * (dist + paddingX));
-            y0 = centerY + ((normY / dist) * (dist + paddingY));
+            let dx = (a * normX) + (c * normY);
```
Comment (SukantPal) [inline packages/canvas/canvas-mesh/src/CanvasMeshRenderer.ts:159]: Please add an explanation of the math/reasoning. For example:

https://github.com/pixijs/pixi.js/blob/cc5f3a1a1662ef69188e199839df57af5cdf9879/packages/core/src/filters/FilterSystem.ts#L600

**Review by SukantPal — CHANGES_REQUESTED** [review]

File: packages/canvas/canvas-mesh/src/CanvasMeshRenderer.ts (hunk 1 above)
Comment (dev7355608) [reply packages/canvas/canvas-mesh/src/CanvasMeshRenderer.ts]: I added some comments and renamed some of the variables to make it easier to follow.

**Review by SukantPal — APPROVED** [review]
Thanks!

**Review by bigtimebuddy — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Verified: https://jsfiddle.net/bigtimebuddy/wke9v1hr/

Thanks @dev7355608!

**Pull request merged**